### PR TITLE
[EN/RU] Update "People/peppy"

### PR DESCRIPTION
**Pull request by Randelman4ik**
added some links

### CODE REVIEW

File: wiki/People/peppy/en.md [hunk 1]
```diff
@@ -9,4 +9,9 @@ tags:
 
 **peppy** (stylised with a lowercase "p") is the online alias of Dean Herbert who is the creator of the osu! project and its primary [developer](/wiki/People/The_Team/Developers).
 
-<!-- TODO: Add links and stuff -->
+## Links
+
+- **[Web site](https://ppy.sh/)**
```
Comment (zeusminus) [inline wiki/People/peppy/en.md:14]:
```suggestion
- **[Website](https://ppy.sh/)**
```
maybe add his GitHub profile as well? 🤔

Comment (Randelman4ik) [reply wiki/People/peppy/en.md:14]: ehm.. idk

Comment (Randelman4ik) [reply wiki/People/peppy/en.md]: i don't think his GH profile is necessary.

Comment (TicClick) [reply wiki/People/peppy/en.md]: I'd still record that as important -- the organization hosts a lot of things

Comment (Randelman4ik) [reply wiki/People/peppy/en.md]: @TicClick  what else can i add?

Comment (TicClick) [reply wiki/People/peppy/en.md]: just that -- I'll add it myself now

**Review by TicClick — APPROVED** [review]

**Pull request merged**